components/dropzone/stories/dropzone.stories.js

```diff
@@ -44,7 +46,33 @@ export default {
 	},
 };
 
-export const Default = Template.bind({});
+export const Default = ({
+	customHeading,
+	customDescription,
+	...args
+}) => {
+	return html`
+		<div>
+			${Template({
+				...args
+			})}
+
+			${isChromatic() ?
```

Comment:
If there is already coverage of the illustrated message wrapping in the illustrated message stories, do we need to add it here as well?

Reply:
Hmm good point. Technically no, but it also doesn't hurt to have it since it will be in the same Chromatic screenshot as the Default, so we won't be charged for it from what I understand. It might give us a little more coverage just to make sure if things in Illustrated Message change, it doesn't affect Dropzone in unexpected/unwanted ways